back/spec/services/avatars_service_spec.rb

```diff
@@ -35,6 +35,25 @@
     end
   end
 
+  describe 'avatars_for_folder' do
+    it 'returns stuff in a folder' do
```

Comment:
```suggestion
    it 'returns avatars for participants in a folder' do
```

Reply:
I forgot to give it a proper name before committing 😅